hivemq-edge/src/frontend/src/components/rjsf/Templates/CompactBaseInputTemplate.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChangeEvent, FC } from 'react'
+import { BaseInputTemplateProps, getInputProps } from '@rjsf/utils'
+import { FormControl, Input } from '@chakra-ui/react'
+
+export const CompactBaseInputTemplate: FC<BaseInputTemplateProps> = (props) => {
+  const {
+    id,
+    type,
+    value,
+    label,
+    schema,
+    onChange,
+    onChangeOverride,
+    onBlur,
+    onFocus,
+    options,
+    required,
+    readonly,
+    rawErrors,
+    autofocus,
+    placeholder,
+  } = props
+  const inputProps = getInputProps(schema, type, options)
+
+  const _onChange = ({ target: { value } }: ChangeEvent<HTMLInputElement>) =>
+    onChange(value === '' ? options.emptyValue : value)
```

Comment:
Are strings made of spaces only valid?
Would you want to trim the input somewhere to avoid bad surprises?

Reply:
i think spaces can be valid. it should be capable of any UTF-8 Character, so I would not recommend trimming

Reply:
I'm also merely replicating the code from the component I'm overriding, in order to maintain consistency